pygenometracks/readBed.py

```diff
@@ -221,31 +174,77 @@ def get_bed_interval(self, bed_line):
                         r = '.'
                 line_values.append(r)
 
-            elif idx in [1, 2, 6, 7, 9]:
-                # start and end fields must be integers, same for thichStart(6),
-                # and thickEnd(7) and blockCount(9) fields
+            elif idx in [1, 2]:
```

Comment:
why don't you need to check the others columns which were previously there?

Reply:
I check them later: 1 and 2 are mandatory to be integer, if they are not I raise an error, whereas 6, 7, 9 are optional.